feature/bgp/policybase/otg_tests/import_export_multi_test/import_export_multi_test.go

```diff
@@ -210,7 +217,9 @@ func configureImportExportMultifacetMatchActionsBGPPolicy(t *testing.T, dut *ond
 			}
 		}
 		communitySetRegex.SetCommunityMember(pd2cs1)
-		communitySetRegex.SetMatchSetOptions(matchAny)
+		if deviations.BGPConditionsMatchCommunitySetUnsupported(dut) {
```

Comment:
This path is required and the purpose of the test is to discover if it is supported.. If the implementation does not support the path, then some alternative may be proposed. 

 I am wondering what your intent is?  This use of deviation doesn't seem to do anything?  It reads line the intent is  bgp community SetMatchSetOptions is "not supported" and then some alternative path or CLI config should be used.     The way the logic is written here, this says if the path is not supported, then try to use the unsupported path (which is what I mean by it's not actually doing anything?).

Reply:
The use of the deviation in existing code at https://github.com/openconfig/featureprofiles/pull/3574/files#diff-a8cc268316b46238349abd13e5dc4211e34da8e024aaa276e0b6c9a57da1dd67R237 is sensible.  There a alternative path (which is deprecated in the public OC models) that is used to achieve the desired results.   That is an sensible and acceptable use of this deviation.

Reply:
(ref to deprecated path: https://github.com/openconfig/public/blob/c58e42f811de0e381f38781179cbcf61e34d3e0d/release/models/bgp/openconfig-bgp-policy.yang#L579)

Reply:
Did you mean to write:

`if !deviations.BGPConditionsMatchCommunitySetUnsupported(dut)`

Reply:
Based on the readme and script, it seems script intent to test -
 `/routing-policy/policy-definitions/policy-definition/statements/statement/conditions/bgp-conditions/match-community-set/config/match-set-options`

In scenario, if this path is not supported, we can use below alternative deprecated path with `if deviations.BGPConditionsMatchCommunitySetUnsupported(dut) {` - 

deprecated path - `/routing-policy/defined-sets/bgp-defined-sets/community-sets/community-set/{config,state}/match-set-options`

Here `communitySetRegex` is based on above deprecated path. 

Above changes are considering - https://github.com/openconfig/public/blob/c58e42f811de0e381f38781179cbcf61e34d3e0d/release/models/bgp/openconfig-bgp-policy.yang#L61-L70

Please correct me if we are missing something with above understanding.

Reply:
Thanks for the reference, now I get it!